css/customstyles.css

```diff
@@ -181,6 +181,7 @@ table th[class*="col-"] {
 
 table td:first-child {
     width: 150px;
```

Comment:
I wonder what was the reason to fix the first column for all tables in this style? My only guess was the tags pages, e.g.  http://samvera.github.io/tag_getting_started.html 
However the fixed column width breaks the view on mobile devices.  I suggest to remove this line.

Reply:
Working on _i18n support for the samvera documentation theme (see https://github.com/jmfrenzel/documentation-theme-jekyll-multiple_language_pop) and at the same time on a local site for research data management at RUB using a fixed table width one creates problems in case breaking long words (as we have in German). 
For now it would be sufficient to get the idea behind introducing the fixed table width as in the original theme https://github.com/tomjoht/documentation-theme-jekyll/ it wasn't used. 

In our dev deploy we use the latter config. Compare the English https://jekyll.rd.ruhr-uni-bochum.de/tag_getting_started.html and the German versions  https://jekyll.rd.ruhr-uni-bochum.de/de/tag_getting_started.html